src/index.js

```diff
@@ -274,6 +278,13 @@ export default class VueI18n {
         )
       }
     }
+
+    if (this._formatFallbackMessages) {
+      const parsedArgs = parseArgs(...values)
+
+      return this._render(key, 'string', parsedArgs.params, key)
+    }
+
```

Comment:
https://github.com/kazupon/vue-i18n/pull/637/files#diff-1fdf421c05c1140f6d71444ea2b27638R266
We need to change from `_warnDefault `, due to have some behavior.

Reply:
I don't exactly understand what you mean. Do you mean we should change the wording here, or?